easybuild/easyconfigs/m/metaWRAP/metaWRAP-1.2.2-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,61 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Tarball'
+
+name = 'metaWRAP'
+local_commit = '2df9b25'
+version = '1.2.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/bxlab/metaWRAP'
+description = """MetaWRAP aims to be an easy-to-use metagenomic wrapper suite that accomplishes the core tasks of
+ metagenomic analysis from start to finish: read quality control, assembly, visualization, taxonomic profiling,
+ extracting draft genomes (binning), and functional annotation."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['https://github.com/bxlab/metaWRAP/archive/']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+checksums = ['67e1d86cafc20a8b2713000d18a169a25b4a7dd7c2d4433b51b2e8ad1d4627ff']
+
+builddependencies = [
+    ('Java', '11', '', True),
+]
+
+# see https://github.com/bxlab/metaWRAP/blob/master/installation/dependancies.md
+dependencies = [
+    ('Python', '2.7.15'),
+    ('Perl', '5.28.1'),
+    ('R', '3.6.0'),  # incl. ggplot2
+    ('SPAdes', '3.13.1'),
+    ('BWA', '0.7.17'),
+    ('MEGAHIT', '1.2.8'),
+    ('QUAST', '5.0.2', versionsuffix),
+    ('SAMtools', '1.9'),
+    ('MetaBAT', '2.14'),
+    ('CONCOCT', '1.1.0', versionsuffix),
+    ('MaxBin', '2.2.7', '-Perl-%(perlver)s'),
+    ('Kraken', '1.1.1', '-Perl-%(perlver)s'),
+    ('KronaTools', '2.7.1'),
+    ('CheckM', '1.0.18', versionsuffix),
+    ('Salmon', '0.14.2'),
+    ('Seaborn', '0.9.0', versionsuffix),
+    ('BLAST+', '2.9.0'),
+    ('Bowtie2', '2.3.5.1'),
+    ('FastQC', '0.11.8', '-Java-11', True),
+    ('Trim_Galore', '0.6.2', '-Java-11', True),
```

Comment:
@Darkless012 You'll need to remove the `True`, since `Trim_Galore` is installed with a non-system toolchain (see #9172)